### ISSUE TITLE
🚀 Feature: Add Slack command bot templates

### ISSUE THREAD

**Issue by Haimantika** [appwrite/templates#228]
Appwrite Functions allow you to extend and customize your Appwrite server functionality by executing your custom code. 🤩 You can learn more at our official [Appwrite Functions docs](https://appwrite.io/docs/functions).

### What you’re building

Your task is to implement the Slack command bot in Node.js.

Slack apps can be created to handle `/` commands by making an HTTP request to an API endpoint. The Slack command bot should be a template to handle basic Slack command bot actions following their [API specifications](https://api.slack.com/interactivity/slash-commands#getting_started). The behavior of the Slack bot should be similar to our [Discord bot](https://github.com/appwrite/templates/tree/main/node/discord-command-bot) and act as a basic `Hello world!` type application.

The starter commands should be simple, but the code should be architected in a way that allows developers to customize and expand the commands supported. Make sure to follow our [contributing guide](https://github.com/appwrite/templates/blob/main/CONTRIBUTING.md).

### Tasks summary:

- [ ]  Fork & clone Appwrite's [templates](https://github.com/appwrite/templates) repository
- [ ]  Create a branch `feat-implement-slack-command-bot-node`
- [ ]  Write code for the function in folder `node/slack-command-bot/`.
- [ ]  Test the function using Appwrite, **provide a video**.
- [ ]  Write a `README.md` explaining what the function does, how to use it, and **customize it**. Here is [an example](https://github.com/appwrite/templates/tree/main/node/whatsapp-with-vonage).
- [ ]  Submit a pull request to our [templates](https://github.com/appwrite/templates) repository

If you need any help, contact us on our [Discord server](https://discord.gg/GSeTUeA).

Are you ready to work on this issue? 🤔 Let us know, and we will assign it to you 😊

**Happy coding!**

**Comment by ManasMadan** [comment]
Hi Please assign this issue to me

**Comment by HrithikSampson** [comment]
Hi Please Assign this issue to me

**Comment by Haimantika** [comment]
> Hi Please assign this issue to me

Hi @ManasMadan  have assigned the issue to you! Thank you for showing interest in contributing to Appwrite! Happy Hacktoberfest 🎃

Notes:

Please update us with your progress every 3 days, so that we know that you are working on it.
Join us on Discord - https://appwrite.io/discord to chat about Hacktoberfest and Appwrite!

**Comment by Haimantika** [comment]
> Hi Please Assign this issue to me

Hi, we are assigning issues on a first-come, first-serve basis, if the other user decides to drop, the issue will go to you.

**Comment by ManasMadan** [comment]
> > Hi Please assign this issue to me
> 
> Hi @ManasMadan have assigned the issue to you! Thank you for showing interest in contributing to Appwrite! Happy Hacktoberfest 🎃
> 
> Notes:
> 
> Please update us with your progress every 3 days, so that we know that you are working on it. Join us on Discord - https://appwrite.io/discord to chat about Hacktoberfest and Appwrite!

Hi this repo is eligible for swags right ? xD

**Comment by Haimantika** [comment]
> > > Hi Please assign this issue to me
> > 
> > 
> > Hi @ManasMadan have assigned the issue to you! Thank you for showing interest in contributing to Appwrite! Happy Hacktoberfest 🎃
> > Notes:
> > Please update us with your progress every 3 days, so that we know that you are working on it. Join us on Discord - https://appwrite.io/discord to chat about Hacktoberfest and Appwrite!
> 
> Hi this repo is eligible for swags right ? xD

I have added the hacktoberfest label, so yes. You will get a swag only after you have raised a PR and it gets the `hacktoberfest-accepted` label

**Comment by Gilbishkosma** [comment]
hi @Haimantika , i can work on this if @ManasMadan drops.

**Comment by ManasMadan** [comment]
@Haimantika A Few Doubts,
- Since slack doesn't have a api to create commands i don't need to work on that
- I need to create a appwrite function that will be triggered when the command is executed on slack right ? and this is what i need to show in the video
Thanks

**Comment by itsmesuraj20** [comment]
Hey @Haimantika please assign this to me

**Comment by ManasMadan** [comment]
@itsmesuraj20 I am working on it. Please find another issue

**Comment by Haimantika** [comment]
> @Haimantika A Few Doubts,
> 
> * Since slack doesn't have a api to create commands i don't need to work on that
> * I need to create a appwrite function that will trigger the command on slack right ? and this is what i need to show in the video
>   Thanks

1. The doc here says that you can use the API for slash commands - https://api.slack.com/interactivity/slash-commands#getting_started
2. Yes

**Comment by ManasMadan** [comment]
> > @Haimantika A Few Doubts,
> > 
> > * Since slack doesn't have a api to create commands i don't need to work on that
> > * I need to create a appwrite function that will trigger the command on slack right ? and this is what i need to show in the video
> >   Thanks
> 
> 1. The doc here should  help you with slash commands - https://api.slack.com/interactivity/slash-commands#getting_started
> 2. Yes

Cool !
I Am Almost done with it. Will Be Creating a PR by 14th

**Comment by ManasMadan** [comment]
@Haimantika I have Created a PR #236 Merging that PR closes this issue
